pages/pipelines/secrets.md.erb

```diff
@@ -155,3 +155,49 @@ steps:
         https://api.github.com/repos/my-org/my-app/deployments
 ```
 {: codeblock-file="pipeline.yml"}
+
+## Two-Factor Authentication
+
+Two-factor authentication (2FA) can be added to provide an additional layer of security to your Buildkite account and make sure you're the only person who has access to handling your builds. Once 2FA is enabled on your Buildkite account, the only way to login to your account is by knowing both your password and a unique code generated by a third-party application such as [1Password], [OTP Auth], [Duo Mobile], [Authy], or [Google Authenticator].
+
+You can set up two-factor authentication in the Buildkite dashboard. To do it, click your profile picture and select Personal Settings in the drop-down menu. 
+
+<%= image "2fa-1.png", width: 710, height: 262, alt: "Screenshot of Personal Settings Button" %>
+
+Next, navigate to the Two-Factor Authentication tab and click it (you may be asked to enter your password in the Confirm Password field).
+<%= image "2fa-3.png", width: 380, height: 238, alt: "Screenshot of Password Confirmation" %>
+<br>
+Enter your Buildkite account password and proceed.
+<%= image "2fa-2.png", width: 850, height: 372, alt: "Screenshot of Accessing Two-Factor Authentication" %>
```

Comment:
Might wanna obscure the email address here?

Reply:
Yep, you're right, thanks!